packages/cli/src/public-api/v1/handlers/tags/tags.handler.ts

```diff
@@ -66,7 +69,7 @@ export = {
 		},
 	],
 	getTags: [
-		globalScope('tag:read'),
+		apiKeyHasScopeWithGlobalScopeFallback({ scope: 'tag:list' }),
```

Comment:
`tag:list` is more correct than `tag:read` and I think it's not breaking because all roles that have `tag:read` currently also have `tag:list`

https://github.com/n8n-io/n8n/blob/d76a2a0eb8dc2857311910b7c2c63ec7a0563e76/packages/cli/src/public-api/permissions.ee/global-roles-scopes.ts#L18-L22

https://github.com/n8n-io/n8n/blob/d76a2a0eb8dc2857311910b7c2c63ec7a0563e76/packages/cli/src/public-api/permissions.ee/global-roles-scopes.ts#L45-L48

Just thinking out loud.